cadence/transactions/evm/create_new_account_with_coa.cdc

```diff
@@ -0,0 +1,64 @@
+import Crypto
+
+import "EVM"
+
+/// Creates a new Flow Address and EVM account with a Cadence Owned Account (COA) stored in the account's storage.
+///
+transaction(
+    key: String,  // key to be used for the account
+    signatureAlgorithm: UInt8, // signature algorithm to be used for the account
+    hashAlgorithm: UInt8, // hash algorithm to be used for the account
+    weight: UFix64, // weight to be used for the account
```

Comment:
My last comment actually leads me to think whether this argument is needed, since we only accept full-weight keys.
Shouldn't we omit this argument and hardcode the weight in line 50 to `1000` ?